stanford/roles/cluster/defaults/main.yml

```diff
@@ -203,7 +236,7 @@ CLUSTER_ELBS:
             to_port: 443
             cidr_ip: '0.0.0.0/0'
       subnets:
-        - "{{ COMMON_DEPLOYMENT }}-subnet-public-{{ CLUSTER_NUMBER }}"
+        - "{{ DEPLOYMENT_SUBNET_PUBLIC_ID }}"
```

Comment:
Since these settings weren't being respected (and `CLUSTER_SUBNET_ID` was being loaded instead),
we didn't catch that `subnets` must be a list of `id`s, not `name`s.